app/assets/javascripts/controllers/notes/notes.js

```diff
@@ -48,6 +48,7 @@ class NotesCtrl extends PureCtrl {
   constructor(
     $timeout,
     $rootScope,
+    $element,
```

Comment:
What's this used for?

Reply:
Whoop, this is a leftover from my first stab at integrating React components with controllers. Will remove!

Reply:
I highly recommend using an editor with built-in ESLint support ;) It's really a must-have today for writing safe code.